doc/source/whatsnew/v0.17.0.txt

```diff
@@ -94,6 +94,13 @@ Other enhancements
 
 - Enable `read_hdf` to be used without specifying a key when the HDF file contains a single dataset (:issue:`10443`)
 
+- :meth:`~pandas.io.read_sas` provides support for reading SAS XPORT format files:
```

Comment:
make this a separate sub-section (use ^^^^^). put a ref so you can refer from the highlites above. Add in a link the actual docs as well (io.sas)